Makefile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 Hewlett Packard Enterprise Development LP
+# Copyright 2022 Hewlett Packard Enterprise Development LP
```

Comment:
It should be 2021-2022, don't drop old years.

Reply:
Fixed.